src/others/sysmon/lv_sysmon.c

```diff
@@ -15,26 +15,16 @@
 #include "../../misc/lv_async.h"
 #include "../../stdlib/lv_string.h"
 #include "../../widgets/label/lv_label.h"
+#include "../../display/lv_display_private.h"
 
 /*********************
  *      DEFINES
  *********************/
-#define MY_CLASS (&lv_sysmon_class)
 
 #define SYSMON_REFR_PERIOD_DEF 300 /* ms */
```

Comment:
This should be

```c
#ifndef SYSMON_REFR_PERIOD_DEF 
#define SYSMON_REFR_PERIOD_DEF 300 /* ms */
#endif


```

Reply:
I don't know why it has marked the old code instead of the new code. But it has. That's annoying.

Reply:
A wasn't planning planning to expose this to keep lv_conf.h smaller, but we have the guard here, so if someone looks in to the code, he will see that it can be overwritten.